internal/server/iigointernal/judiciary_test.go

```diff
@@ -0,0 +1,309 @@
+package iigointernal
+
+import (
+	"fmt"
```

Comment:
can you run `goimport`?

Reply:
Not sure how to run the command from goland, I've fixed the imports so that the linter is less upset